src/tests/JIT/Regression/JitBlue/Runtime_110958/Runtime_110985.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Optimize>True</Optimize>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="$(MSBuildProjectName).cs" />
+
+    <CLRTestEnvironmentVariable Include="DOTNET_TieredCompilation" Value="0" />
+    <CLRTestEnvironmentVariable Include="DOTNET_JitStress" Value="2" />
+    <CLRTestEnvironmentVariable Include="DOTNET_JitRandomEdgeCounts" Value="1" />
```

Comment:
I'm not sure if this is still true, but I think you need `<RequiresProcessIsolation>true</RequiresProcessIsolation>` under `<PropertyGroup>` for `CLRTestEnvironmentVariable` to work.